client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js

```diff
@@ -37,61 +37,87 @@ function LaptopForm() {
 			}),
 			headers: { "content-type": "application/json" },
 		});
-	}
+	};
+
+	const modale = document.querySelector('.modale-content');
+	const form = document.querySelector('.form');
+	
+
+	//function display greeting after submit the from
 	function messageAlert() {
-		alert(
-			"thank you for your completing the request form. your request have been recived "
-		);
-	}
+		document.body.style.backgroundColor = 'black';
```

Comment:
When writing React, there should be no modification (such as changing styles as here) of elements in the DOM that are defined and controlled by React - this should happen through React itself, otherwise unexpected things will happen. Instead of doing calls such as `document.querySelector('.classname')`, there should be rendering of the React component here (the laptop form) based on a change of state. I also would advise against hiding the form using CSS - the form would be better being conditionally rendered depending on whether state has changed (this could be from a new piece of local state using another `useState` hook).

Reply:
thank you shieldo, for your feedback i understood